src/util/allocate_objects.cpp

```diff
@@ -147,9 +147,10 @@ exprt allocate_objectst::allocate_dynamic_object(
   if(allocate_type.id() == ID_empty)
   {
     // make null
-    null_pointer_exprt null_pointer_expr(to_pointer_type(target_expr.type()));
-    code_assignt code(target_expr, null_pointer_expr);
-    code.add_source_location() = source_location;
+    const code_assignt code{
```

Comment:
Don't mark this one `const`, and instead use `output_code.add(std::move(code));` below.

Reply:
Done :+1:

Reply:
Thanks! But now clang-format would ask for a different formatting it seems...

Reply:
Fixed it and CI passed so I will merge this PR.